Describe the Interpolated line symbol layer (#6998)
qgis/QGIS-Documentation: docs/user_manual/style_library/symbol_selector.rst

  shapeburst to render the arrow body. Combined with the geometry generator,
  this type of layer symbol helps you representing flow maps.
* **Geometry generator** (see :ref:`geometry_generator_symbol`)
* **Interpolated line**: allows to render a line whose :guilabel:`Stroke width`

>   and/or :guilabel:`Color` may be constant (given a :guilabel:`Width` and
>   :guilabel:`Color` parameters) or vary along the road.

Author: ```suggestion
  and/or :guilabel:`Color` may be constant (given a :guilabel:`Fixed width` and
  :guilabel:`Single color` parameters) or vary along the geometry.
```

>   When varying, the width and color of the line are interpolated along the
>   geometry using feature attributes (or an expression) with a range of values.
>   Necessary inputs are:

Author: ```suggestion
  When varying, necessary inputs are:
```

  * :guilabel:`Start value` and :guilabel:`End value`: XXX
  * :guilabel:`Min. value` and :guilabel:`Max. value`: XXX
    Press the |refresh| :sup:`Load` button to automatically fill them based
    on start or end values expressions.
  * Only available for the stroke option:

>     * :guilabel:`Min. width` and :guilabel:`Max. width`: define how wide

Reviewer: ```suggestion
    * :guilabel:`Min. width` and :guilabel:`Max. width`: define the range of the varying width. `Min. Width` corresponds to `Min. Value` and `Max. Width` corresponds to `Max. Value`
```

>       the line can be, linearly scaling the range of values onto XXX.

Reviewer: ```suggestion
```

      A :ref:`unit <unit_selector>` can be associated.

>     * |checkbox| :guilabel:`Use absolute value`: XXX

Reviewer: ```suggestion
    * |checkbox| :guilabel:`Use absolute value`: only consider absolute value for interpolation (negative value are use as positive).
```

>     * |checkbox| :guilabel:`Ignore out of range`: XXX

Reviewer: ```suggestion
    * |checkbox| :guilabel:`Ignore out of range`: value out of the range `Min. Value`/`Max. Value` are not rendered.
```
